sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/MetadataReferences/CoreMetadataReference.cs

```diff
@@ -0,0 +1,134 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2020 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+#if !NETFRAMEWORK
```

Comment:
do you plan to have common libraries between the CoreMetadataRefrence and FrameworkMetadataReference in a base class? currently, I see duplication e.g. `Mscorlib`

Reply:
I think it would be too much of a change to do this since static classes cannot derive from other classes than Object

Reply:
ah indeed

well, we'll see if we need to change this when writing more rules.